pkgs/development/compilers/gcc/10/default.nix

```diff
@@ -7,7 +7,7 @@
 , profiledCompiler ? false
 , langJit ? false
 , staticCompiler ? false
-, enableShared ? true
+, enableShared ? !stdenv.hostPlatform.isStatic
```

Comment:
Don't do this!!! We want the toolchain to be able to produce both static & shared. Otherwise we get mass rebuilds for no reason. GCC & LLVM should be capable of producing both shared & static binaries as we need them.

Reply:
OK not doing this for now.